frontend/app/routes/_gcweb-app.personal-information._index.tsx

```diff
@@ -91,14 +99,15 @@ interface PersonalInformationSectionProps {
   children: ReactNode;
   footer?: ReactNode;
   title: ReactNode;
+  icon?: string;
 }
 
-function PersonalInformationSection({ children, footer, title }: PersonalInformationSectionProps) {
+function PersonalInformationSection({ children, footer, title, icon }: PersonalInformationSectionProps) {
   return (
     <section className="panel panel-info !m-0 flex flex-col">
       <header className="panel-heading">
         <h2 className="h3 panel-title">
-          <span className="glyphicon glyphicon-envelope pull-right" aria-hidden="true"></span>
+          <span className={`glyphicon ${icon} pull-right`} aria-hidden="true"></span>
```

Comment:
Use logical AND operator and `clsx` package.

```suggestion
          {icon && <span className={clsx('glyphicon', icon, 'pull-right')} aria-hidden="true"></span>}
```